docs/pages/governance.mdx

```diff
@@ -0,0 +1,50 @@
+---
+description: Turbo by Vercel is the next-generation build system for the web.
+hidden: true
+headeronly: true
+container: true
+searchable: false
+---
+
+# Governance
+
+- Turborepo was created by the team at [Vercel](https://vercel.com) in 2020.
+- Turbopack was created by the team at [Vercel](https://vercel.com) in 2022.
+
+The research and development of Turbo is led by the core team working full-time at Vercel. Turbo is a collaborative open-source effort to build tools for web developers.
+
+## Teams
+
+**Turborepo Lead**: Greg Soltis
+
+**Turbopack Lead**: Tobias Koppers
+
+**Core Team**: Turborepo and Turbopack teams at [Vercel](https://vercel.com) manage the direction of Turbo and leadership of the project.
+
+**Documentation**: DevRel teams at [Vercel](https://vercel.com) manage documentation and learning platforms.
+
+## Roadmap and RFC Process
+
+Large architecture decisions and features start as a [Request for Comments (RFC)](https://github.com/vercel/turbo/discussions/categories/rfc) inside GitHub Discussions. The entire Turbo community is invited to be involved in the process and discuss the tradeoffs of proposed work.
+
+This process is essential to ensure features are built together with the community and our collaborators. More minor changes including bug fixes and documentation improvements are still implemented through GitHub pull requests. However, for larger initiatives, we believe it's essential to follow the RFC process.
+
+## Stability and release phases
+
+- **Experimental**: Experimental features lack essential capabilities required to be ready for stability. The feature is considered to be under active development and is being built to transition into Beta.
+- **Beta**: Beta means the feature does not yet meet our quality standards for a stable release. There is also usually a need for more information or feedback from Turbo developers to validate this feature works as expected in a variety of environments.
+- **Stable**: When the features reach stability, it is considered ready for use in production. APIs marked as stable will only change in a backward-compatible way unless there is a major release.
+- **Deprecated**: A feature is in the process of being removed. Deprecated features include documentation for existing users with the next steps and information on when to expect the feature to be removed.
+
+## Turbo Support
```

Comment:
Should we mention `vercel.com/solutions/turborepo` here anywhere? It's a great enterprise landing page, but doesn't get enough hooks imo.